### ISSUE TITLE
Feature request: cutting lines in Clipping plane

### ISSUE THREAD

**Issue by pgilfernandez** [realthunder/FreeCAD_assembly3#728]
Hi,

The new "solid mode" in Clipping plane is great, thanks!
Would it be possible to add an option to draw the lines  in that solid plane? sometimes it is still hard to differentiate what is the part and what is the cutted plane...

Example:
<img width="512" alt="clipping_plane" src="https://user-images.githubusercontent.com/5942369/125257091-5aa07300-e2fd-11eb-8d9f-18a87fc842fe.png">

Thanks

**Comment by realthunder** [comment]
Yes, it is possible, but it is currently not possible to do it only for the clipping plane. This will enable outline drawing on all faces. That's why it is only enabled in `Hidden line` draw style. Besides, the outline drawing is not very efficient at the moment. You may experience some slow down. You can simply enable `Invert fill color` to make the clip plane stand out.

![Screenshot from 2021-07-12 20-01-01](https://user-images.githubusercontent.com/1207888/125284452-0a043680-e34c-11eb-939b-aabf683045e0.png)

**Comment by herrgerd** [comment]
> 
> 
> Hi,
> 
> The new "solid mode" in Clipping plane is great, thanks! (...)
> Thanks

**Indeed it is!** Had to use Inventor a lot lately and clipping parts and especially assemblies is a nightmare there. Still using some strange blurred "pattern" which makes recognizing details impossible at times. You have to create your own pattern and import it as .bmp to change that. No solid color sectional view possible via on-board means, in a multi thousand $ software suite, go figure :)

**Comment by mwlandolsi** [comment]
Another enhancement, is  to define for every part or body it's hashing style, it could be easier for every surface. This is very important in mechanical engineering.

**Comment by OxygenLiu** [comment]
A noob question, how to activate the Solid Clipping Plane?

**Comment by mwlandolsi** [comment]
@OxygenLiu  go to **View> Clipping plane** menu.

**Comment by OxygenLiu** [comment]
@mwlandolsi no option for solid clipping plane as shown below 

![Screenshot from 2021-07-30 13-03-50](https://user-images.githubusercontent.com/9315996/127603421-bfe3df4a-23dd-43dd-971e-f4aef9763241.png)

FreeCAD version:

OS: Pop!_OS 20.04 LTS (pop:GNOME/pop)
Word size of OS: 64-bit
Word size of FreeCAD: 64-bit
Version: 2021.717.24301 +3627 (Git) AppImage
Build type: Release
Branch: LinkStage3
Hash: f63a5ea7b459872a3798027df0ff25e776524859
Python version: 3.9.6
Qt version: 5.12.9
Coin version: 4.0.1
OCC version: 7.5.2
Locale: English/United States (en_US)

**Comment by realthunder** [comment]
It is only available in the `Daily` branch.
